examples/mongodbatlas_encryption_at_rest_private_endpoint/azure/versions.tf

```diff
@@ -0,0 +1,14 @@
+terraform {
+  required_providers {
+    mongodbatlas = {
+      source  = "mongodb/mongodbatlas"
+      version = "~> 1.18"
```

Comment:
1.19?

Reply:
we currently cannot define a version that is not yet released: https://jira.mongodb.org/browse/CLOUDP-269863

Reply:
aah right